package.json

```diff
@@ -24,10 +25,14 @@
     "@babel/node": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/register": "^7.5.5",
+    "coveralls": "^3.0.6",
     "cross-env": "^5.2.0",
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.18.2",
-    "nodemon": "^1.19.1"
+    "istanbul": "^0.4.5",
```

Comment:
I don't think we need `instanbul` if we're using `nyc` with `coverallls`. Please confirm this.

Reply:
> I don't think we need `instanbul` if we're using `nyc` with `coverallls`. Please confirm this.

@eokwukwe it's very true, I had to confirm that.

Reply:
@eokwukwe i've made the changes.